tests/components/flexit_bacnet/test_config_flow.py

```diff
@@ -94,7 +93,9 @@ async def test_flow_fails(
     assert len(mock_setup_entry.mock_calls) == 1
 
 
-async def test_form_device_already_exist(hass: HomeAssistant, flow_id: str) -> None:
+async def test_form_device_already_exist(
+    hass: HomeAssistant, flow_id: str, mock_flexit_bacnet
```

Comment:
you can also use the mock_config_entry fixture here and add that to hass instead of creating a new one